src/tools/dev/scripts/bv_support/bv_zlib.sh

```diff
@@ -57,6 +57,16 @@ function bv_zlib_host_profile
     fi
 }
 
+function bv_zlib_initialize_vars
+{
+    if [[ "$DO_ZLIB" == "yes" ]] ; then
```

Comment:
In theory, `DO_ZLIB` should only ever be `yes` now, so do we still need to define/use it?